python/nano/src/bigdl/nano/pytorch/k8s/bigdl-submit.py

```diff
@@ -0,0 +1,109 @@
+
+import subprocess
+import sys
+import os
+from uuid import uuid4
+from kubernetes import client, config
+from argparse import REMAINDER, ArgumentParser
+
+def get_args_parser() -> ArgumentParser:
+    """Helper function parsing the command line options."""
+
+    parser = ArgumentParser(description="Nano Training Launcher")
+
+    parser.add_argument(
+        "--nnodes",
+        type=int,
+        default=1,
+        help="Number of nodes",
+    )
+
+    parser.add_argument(
+        "--image",
+        type=str,
+        default="10.239.45.10/arda/nano-run:latest",
```

Comment:
Use external image maybe better

Reply:
I think this should be a image provided by bigdl. Temporarily changing a image in my registry.